harmony/http.py

```diff
@@ -302,21 +336,28 @@ def download(config, url: str, access_token: str, data, destination_file, user_a
     start_time = datetime.datetime.now()
     logger.info(f'timing.download.start {url}')
 
+    _preprocess_download_kwargs(download_kwargs, logger)
+
     if access_token is not None and _valid(config.oauth_host, config.oauth_client_id, access_token):
-        response = _download(config, url, access_token, data, user_agent)
+        response = _download(config, url, access_token, data, user_agent, stream=download_kwargs["stream"])
```

Comment:
See prior comments on `download_kwargs`, but if any kwargs other than "stream" exist, they won't get passed along here.  I'd probably shy away from using kwargs for this and just have a `stream` argument, reserving kwargs for when we can't predict or don't care what the arguments are called.